dev/com.ibm.ws.messaging.common/src/com/ibm/ws/sib/mfp/impl/SchemaSet.java

```diff
@@ -362,6 +363,10 @@ public String toVerboseString() {
     return buf.toString();
   }
 
+  @Override
+  public String toTraceString() {
+      return toVerboseString();
```

Comment:
This is a behavior change to call toVerboseString instead of toString.  Any reason?

Reply:
My thought was that it was more serviceable, but I have changed it to return `toString` instead to continue the previous behavior.